x/concentrated-liquidity/simulation/sim_msgs.go

```diff
@@ -289,6 +295,14 @@ func getRandomTickPositions(sim *osmosimtypes.SimCtx, minTick, maxTick int64, ti
 	return lowerTick, upperTick, nil
 }
 
+func RandomMinAmount(sim *osmosimtypes.SimCtx, token0Desired, token1Desired sdk.Int) (sdk.Int, sdk.Int) {
+	rand := sim.GetRand()
+	percent := sdk.NewDec(int64(sdkrand.RandIntBetween(rand, 0, 100) / 100))
+	minAmount0 := sdk.NewDecFromInt(token0Desired).Mul(percent).TruncateInt()
+	minAmount1 := sdk.NewDecFromInt(token1Desired).Mul(percent).TruncateInt()
```

Comment:
I could be misreading this, but isn't this multiplying by an integer representation of the percent rather than the dec? So for instance, if the random number was 100 to represent 100 percent, it would actually be multiplying the tokenDesired by 100x rather than 1x?

Reply:
but we divide it by 100 at the end right?

Reply:
its random dec in range [0, 1]